src/main.ts

```diff
@@ -20,6 +20,7 @@ pipe(
       retryAfterMs: 1000,
       downloadDocumentURL: config.server.downloadDocumentURL,
       sampleStaticPdfFileName: 'sample.pdf',
+      notificationPrice: 100,
```

Comment:
At some point, we should make these variable editable from system environment :innocent:

Reply:
Yep, there are few params we should let them be editable, so IMHO, I would create a separate PR for all of them